__init__.py

```diff
@@ -25,18 +25,23 @@
 # LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
 # NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE,  EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-
-from typing import Optional
+import datetime
+from datetime import timedelta
 from enum import Enum
-from adapt.intent import IntentBuilder
+from os.path import expanduser
 from random import randint
+from typing import List, Optional
+
+from adapt.intent import IntentBuilder
```

Comment:
we are phasing out adapt imports, it is recommended to import from ovos_workshop.intents instead

adapt only needs to be installed in core, not needed for standalone usage of the skill